glove2word2vec.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016 Manas Ranjan Kar <manasrkar91@gmail.com>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+"""
+CLI USAGE: python glove2word2vec.py -input <GloVe vector file> -output <Output model file>
+
+Convert GloVe vectors into word2vec C format;
+
+model = gensim.models.Word2Vec.load_word2vec_format('/tmp/vectors.txt', binary=False)  # C text format
+
+word2vec embeddings start with a line with the number of lines (tokens?) and the number of dimensions of the file. This allows gensim to allocate memory 
+accordingly for querying the model. Larger dimensions mean larger memory is held captive. Accordingly, this line has to be inserted into the GloVe 
+embeddings file.
+"""
+
+import re
+import sys
+import gensim
+import logging
+import argparse
+import smart_open
+
+logger = logging.getLogger(__name__)
+
+def glove2word2vec(glove_vector_file, output_model_file):
+    """Convert GloVe vectors into word2vec C format"""
+    
+    def get_info(glove_file_name):
+        """ 
+        Function to calculate the number of lines and dimensions of the GloVe vectors to make it Gensim compatible
+        """
+        num_lines = sum(1 for line in smart_open.smart_open(glove_vector_file))
+        if 'twitter' in glove_file_name:
```

Comment:
Done, I have created a function to do the following;

1. Read the first 100 lines of the embeddings file, and count number of float items
2. Appends it to a list
3. Calculate median to finalize the number of `dims`